packages/components/input/README.md

```diff
@@ -0,0 +1,131 @@
+<h1 align="center">
+    <a href="https://tradeshift.com/">
+      <img alt="Tradeshift" src="https://tradeshift.com/wp-content/themes/Tradeshift/img/brand/logo-black.png"/>
+    </a>
+</h1>
+
+<h1 align="center">Elements - input</h1>
+
+<p align="center">
+  Part of the reusable Tradeshift UI Components as Web Components.
+    <a href="https://tradeshift.github.io/elements/?path=/story/ts-input--default">
+      Demo
+    </a>
+</p>
+
+<p align="center">
+    <a href="https://www.npmjs.com/package/@tradeshift/elements.input">
+      <img alt="NPM Version" src="https://badgen.net/npm/v/@tradeshift/elements.input" height="20"/>
+    </a>
+    <a href="https://npmcharts.com/compare/@tradeshift/elements.input?minimal=true">
+		  <img alt="Downloads per month" src="https://badgen.net/npm/dm/@tradeshift/elements.input" height="20"/>
+		</a>
+		<a href="https://www.npmjs.com/browse/depended/@tradeshift/elements.input">
+		  <img alt="Dependent packages" src="https://badgen.net/npm/dependents/@tradeshift/elements.input" height="20"/>
+		</a>
+</p>
+
+## ➤ How to use it
+
```

Comment:
Can you please add slots, props and events here? Also please add how they should be used (adding the input as a slotted node) and the reason for it.